picard/formats/vorbis.py

```diff
@@ -333,28 +333,11 @@ def _info(self, metadata, file):
         metadata['~format'] = self.NAME
 
 
-def _select_ogg_type(filename, options):
-    """Select the best matching Ogg file type."""
-    fileobj = file(filename, "rb")
-    results = []
-    try:
-        header = fileobj.read(128)
-        results = [
-            (option._File.score(filename, fileobj, header), option.__name__, option)
-            for option in options]
-    finally:
-        fileobj.close()
-    results.sort()
-    if not results or results[-1][0] <= 0:
-        raise mutagen.ogg.error("unknown Ogg audio format")
-    return results[-1][2](filename)
-
-
 def OggAudioFile(filename):
     """Generic Ogg audio file."""
     options = [OggFLACFile, OggSpeexFile, OggVorbisFile]
-    return _select_ogg_type(filename, options)
-
+    _File = None
```

Comment:
Oops, I added it(the variable _File) before and forgot to remove it, it is not included while testing options in guess_format anyways.

I'll remove it.